constraints.md

```diff
@@ -52,20 +52,81 @@ Query OK, 1 row affected (0.03 sec)
 
 ## CHECK
 
-TiDB parses but ignores `CHECK` constraints. This is MySQL 5.7 compatible behavior.
+A `CHECK` constraint is used to restrict the values of a field in a table to meet your specified conditions. When the `CHECK` constraint is added to a table, TiDB checks whether the constraint is satisfied during the insertion or updates of data into the table. If the constraint is not met, an error is returned.
 
-For example:
+The syntax for the `CHECK` constraint in TiDB is the same as that in MySQL:
 
 ```sql
-DROP TABLE IF EXISTS users;
-CREATE TABLE users (
- id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
- username VARCHAR(60) NOT NULL,
- UNIQUE KEY (username),
- CONSTRAINT min_username_length CHECK (CHARACTER_LENGTH(username) >=4)
-);
-INSERT INTO users (username) VALUES ('a');
-SELECT * FROM users;
+[CONSTRAINT [symbol]] CHECK (expr) [[NOT] ENFORCED]
+```
+
+Syntax explanation:
+
+- `[]`: the content within `[]` is optional.
+- `CONSTRAINT [symbol]`: specifies the name of the `CHECK` constraint.
+- `CHECK (expr)`: specifies the constraint condition, where `expr` needs to be a boolean expression. For each row in the table, the calculation result of this expression must be one of `TRUE`, `FALSE`, or `UNKNOWN` (for `NULL` values). If the calculation result is `FALSE` for a row, it indicates that the constraint is violated.
+- `[NOT] ENFORCED`: specifies whether to implement the constraint check. You can use it to enable or disable a `CHECK` constraint.
+
+### Add `CHECK` constraints
+
+In TiDB, you can add a `CHECK` constraint to a table using either the [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md) or the [`ALTER TABLE`](/sql-statements/sql-statement-modify-column.md) statement.
+
+- Example of adding a `CHECK` constraint using the `CREATE TABLE` statement:
+
+    ```sql
+    CREATE TABLE t(a INT CHECK(a > 10) NOT ENFORCED, b INT, c INT, CONSTRAINT c1 CHECK (b > c));
+    ```
+
+- Example of adding a `CHECK` constraint using the `ALTER TABLE` statement:
+
+    ```sql
+    ALTER TABLE t ADD CONSTRAINT CHECK (1 < c);
+    ```
+
+When adding or enabling a `CHECK` constraint, TiDB validates the existing data in the table. If any data violates the constraint, the operation of adding the `CHECK` constraint will fail and return an error.
```

Comment:
```suggestion
When adding or enabling a `CHECK` constraint, TiDB checks the existing data in the table. If any data violates the constraint, the operation of adding the `CHECK` constraint will fail and return an error.
```

Reply:
Keep it consistent with "When the `CHECK` constraint is added to a table, TiDB **checks** whether the constraint is satisfied during the insertion or updates of data into the table. "